src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -23,6 +23,9 @@ const StyledCheckbox = styled(Checkbox)`
         height: 20px;
         width: 20px;
     }
+    > span {
+        font-size: 16px;
```

Comment:
Can we use a value from `theme` here? Above I didn't because the checkbox is specifically 20px (and won't change)

Reply:
Yep! I meant to go back and change that but totally forgot